### DISCUSSION TITLE
ask for Accessibility every time

### DISCUSSION THREAD

**Discussion by Abdulkader-Safi** [MonitorControl/MonitorControl#1527] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

every time I start my Mac MonitorControl asks me for access to the Accessibility in sittings 
but its already checkedit

### Steps to reproduce

restart my mac

### Expected behavior

just open

### Environment Information (please complete the following information)

```markdown
- macOS version: Sonoma 14.2.1
- Mac model: MacBook Pro M2 Pro 16-inch
- MonitorControl version: Version 4.2.0 Build 7048
- Monitor(s):
- Apple Silicon/M1 (yes or no): yes M2 Pro
```

**Comment by waydabber** [comment, marked as answer]
You might need to reset the accessibility database (it can get corrupted sometimes). Google `tccutil`. :)

**Reply by Abdulkader-Safi** [reply]
Thanks so much, that fix my issue
